website/public/locales/en.json

```
@@ -41,19 +41,19 @@
     "subtitleChannel": "The cautious should send the decryption key in a separate communication channel.",
     "rowLabelOneClick": "One-click link",
     "rowLabelShortLink": "Short link",
-    "rowLabelDecryptionKey": "Decryption Key"
+    "rowLabelDecryptionKey": "Decryption key"
   },
   "secret": {
     "titleFile": "File downloaded",
-    "titleMessage": "Decrypted Message",
+    "titleMessage": "Decrypted message",
     "subtitleMessage": "This secret might not be viewable again, make sure to save it now!",
     "buttonCopy": "Copy"
   },
   "delete": {
     "buttonDelete": "Delete",
     "messageDeleted": "The secret is removed from the server!",
-    "dialogTitle": "Delete the Secret?",
-    "dialogMessage": "Are you sure you want to remove this Secret?",
+    "dialogTitle": "Delete the secret?",
```

```suggestion
    "dialogTitle": "Delete secret?",
```

Does this read nicer?